### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for items 24025, 24033, 24078, 24079, 24111, 24112, 24113, 24114, 24208, 24210, 24214, 24226, 24231, 24232, 24247, 24249, 24252

https://divine-pride.net/database/item/24025
https://divine-pride.net/database/item/24033
https://divine-pride.net/database/item/24078
https://divine-pride.net/database/item/24079
https://divine-pride.net/database/item/24111
https://divine-pride.net/database/item/24112
https://divine-pride.net/database/item/24113
https://divine-pride.net/database/item/24114
https://divine-pride.net/database/item/24226
https://divine-pride.net/database/item/24231
https://divine-pride.net/database/item/24232
https://divine-pride.net/database/item/24247
https://divine-pride.net/database/item/24249
https://divine-pride.net/database/item/24252
``````
update script
``````

https://divine-pride.net/database/item/24208
``````
remove price buy
update script
``````

https://divine-pride.net/database/item/24210
https://divine-pride.net/database/item/24214
``````
remove price buy
``````

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -96071,10 +96071,10 @@ Body:
     Refineable: true
     Script: |
       bonus bMaxHP,20;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:96073]: Aegis data
```suggestion
      bonus bMaxHP,20+10*getrefine();
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -96165,6 +96165,7 @@ Body:
     EquipLevelMin: 1
     Refineable: true
     Script: |
+      bonus bMatk,10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:96168]: Aegis data
```suggestion
      .@r = getrefine();
      bonus bBaseAtk,.@r;
      bonus bMatk,10+.@r;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -98583,9 +98583,9 @@ Body:
       .@r = getrefine();
       bonus bAtk2,(5+.@r);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98584]:
```suggestion
      bonus bBaseAtk,(5+.@r);
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -98668,9 +98668,7 @@ Body:
     Script: |
       bonus bFlee,5;
       .@r = getrefine();
-      if (.@r>=2) {
-         bonus bCritical,2;
-      }
+      bonus bFlee,.@r/2;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98671]: Delete the previous bFlee,5 and
```suggestion
      bonus bFlee,5+.@r/2;
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -98688,9 +98686,7 @@ Body:
     Script: |
       bonus bCritical,5;
       .@r = getrefine();
-      if (.@r>=2) {
-         bonus bCritical,2;
-      }
+      bonus bCritical,.@r/2;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98689]: Delete the previous crit 5
```suggestion
      bonus bCritical,5+.@r/2;
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -98876,7 +98872,7 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus2 bSkillUseSPrate,"MC_MAMMONITE",20+(3*.@r);
+      bonus2 bSkillAtk,"MC_Mammonite",20+(5*.@r);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98875]: Aegis
```suggestion
      bonus bMaxHP,.@r*10;
      bonus2 bSkillAtk,"MC_Mammonite",20+(5*.@r);
```

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -98898,9 +98894,9 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus2 bSkillUseSPrate,"MG_COLDBOLT",20+(3*.@r);
-      bonus2 bSkillUseSPrate,"MG_FIREBOLT",20+(3*.@r);
-      bonus2 bSkillUseSPrate,"MG_LIGHTNINGBOLT",20+(3*.@r);
+      bonus2 bVariableCastrate,"MG_COLDBOLT",-20-(5*.@r);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98897]: Aegis data
```suggestion
      bonus bMaxHP,.@r*10;
      bonus2 bVariableCastrate,"MG_COLDBOLT",-20-(5*.@r);
```

File: db/re/item_db_equip.yml [hunk 8]
```diff
@@ -98932,7 +98928,6 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus bMaxHP,.@r*10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:98935]: The bonus is here on aegis data but yeah it's missing on the description.. don't remove it

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:96073]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:96168]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:98584]: Thanks

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:98671]: Thanks

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:98689]: Thanks

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:98875]: Thanks

File: db/re/item_db_equip.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:98897]: Thanks

File: db/re/item_db_equip.yml (hunk 8 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**